### ISSUE TITLE
Perform L7: build a live camera app

### ISSUE THREAD

**Issue by pvhee** [pvhee/deeplearning_experiments#1]
1) [x] Use notMNIST and concatenate images into sequences of up to 5, then train DNN to recognise sequences. _Update: Reached 0.934 test accuracy after introducing dropout (0.25), see https://github.com/pvhee/deeplearning_experiments/commit/1a0a5e19fdb5b31ac8cad29be5b91ab4614fe423_
2) [x] Use real data (SVHN dataset)
3) [x] Create app (React Native?) -- first version at https://github.com/pvhee/digit-recognizer-app

**Comment by pvhee** [comment]
Examples on SVHN: 
* https://github.com/nate-parrott/juypter-notebooks/blob/master/svhn-keras.ipynb
* Published base line model http://static.googleusercontent.com/media/research.google.com/en//pubs/archive/42241.pdf and blog post in MIT Tech Review https://www.technologyreview.com/s/523326/how-google-cracked-house-number-identification-in-street-view/

**Comment by pvhee** [comment]
First results on artificial dataset (20.000 imgs): 
- Validation accuracy: 0.9823
- Test accuracy: 0.928
